Libraries/LibWeb/Layout/AnonymousImageBox.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2025, Bohdan Sverdlov <none@none.org>
```

Comment:
Please use a real email address for copyright. If you're concerned about anonymity, you should probably know that your commits already include an email - take a look at https://github.com/LadybirdBrowser/ladybird/commit/1767f799e4ec8dfef249d7a70256634defa3a9b3.patch